app/models/work.rb

```diff
@@ -9,7 +9,7 @@ class Work < ApplicationRecord
   has_many :contributors, dependent: :destroy
   has_many :related_links, dependent: :destroy
   has_many :related_works, dependent: :destroy
-  has_many_attached :files
+  has_many :attached_files, dependent: :destroy
```

Comment:
Can you say more about this change?

Reply:
Nevermind, I now see it's so you can hang the label and hide attrs off of attached files.